opennlp-tools/src/test/java/opennlp/tools/ml/perceptron/PerceptronPrepAttachTest.java

```diff
@@ -89,4 +91,36 @@ public void testPerceptronOnPrepAttachDataWithStepSizeDecrease() throws IOExcept
 
     testModel(model, 0.7791532557563754);
   }
+
+  @Test
+  public void testModelSerialization() throws IOException {
+
+    Map<String, String> trainParams = new HashMap<String, String>();
```

Comment:
yes, lets just refactor this throughout the codebase and I will of course update my additions here